packages/x-components/src/wiring/wires.factory.ts

```diff
@@ -144,13 +145,16 @@ export function wireDispatchWithoutPayload(action: string): AnyWire {
  * @returns A factory to create wires that invoke the service methods.
  * @public
  */
-export function wireService<SomeService>(service: SomeService): WireService<SomeService> {
+export function wireService<SomeService>(
+  service: SomeService & SubObject<SomeService, MonadicFunction>
```

Comment:
`SubObject<SomeService, MonadicFunction>` as it name says it should be a sub-object of the `SomeService` generic parameter. I'm using it here to try to add to TypeScript enough information so it can understand the below `service[method]` call.